server/tests/integration/graphql.test.js

```diff
@@ -298,6 +299,12 @@ describe('graphql', () => {
                                 test {
                                     id
                                 }
+                               # atVersion {
```

Comment:
I'd like to put these changes in another PR.

Reply:
Perhaps we can remove the commented lines here for now then?

Reply:
I put a TODO there